src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Index.txt

```diff
@@ -0,0 +1,20 @@
+>> Index([1, 2, 3, 4, 5], 3)
```

Comment:
>[1, 2, 3, 4, 5] [](http://example.com/codeflow?start=9&length=15)

nit: 

With this case, it's ambiguous whether the result of 3 was from the array or just taking the 2nd arg. 

Consider a test case like this to be more assertive:
>> Index([10, 20, 30, 40, 50], 3)
{ Value :30}

Reply:
Done